example/enzimology_calculations.cpp

```diff
@@ -0,0 +1,52 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2023 Meskauskas Audrius
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include <mp-units/systems/enzimology/enzimology.h>
+#include <mp-units/ostream.h>
+#include <mp-units/quantity.h>
+#include <mp-units/quantity_point.h>
+#include <iostream>
+
+using namespace mp_units;
+using namespace mp_units::si::unit_symbols;
+using namespace mp_units::si;
+
+int main()
+ {
+    // In the test tube with 2 ml of test solution, 10 micro-molles of the substrate have been processed in 5 minutes.
+    auto time = 5. * min;
+    auto processed_substrate = 10. * umol;
+    auto volume = 2. * milli<litre>;
+
+    // 1 unit (U) is the amount of enzyme that catalyses the reaction of 1 µmol of substrate per minute
+
+    auto units = processed_substrate / time;
+    quantity<enzimology::nanokatal> nanokatals = units; // conversion if we want, 1 U is about 16.67 nkat
+
+    auto activity = units / volume;
+
+    auto protein_concentration = (20.0 * mg) / (1.0 * milli<litre>);
+    quantity<enzimology::specific_activity> specific_activity = activity / protein_concentration;
```

Comment:
I am not an expert in the enzymology domain in any sense, but reading the example and Table 1 form the document you referenced, I think it might be incorrect. It seems that `activity` and `specific_activity` are not units but quantity specifications? Similarly, `speed` is not a unit of `m / s`. It is the quantity of speed measured in `m / s`. Do you say "I measured `5 activities`" or do you say "I measured the activity of `5 U/ml`"?

If that is true, then the above should look like one of the below:

```suggestion
    quantity<enzimology::catalytic_activity[umol / min]> units = processed_substrate / time;
    quantity<enzimology::catalytic_activity[nkat]> nanokatals = units; // conversion if we want, 1 U is about 16.67 nkat
    
    quantity<enzimology::volume_activity[U / ml]> activity = units / volume;

    quantity<enzimology::protein_concentration[mg / ml]> protein_concentration = (20.0 * mg) / (1.0 * ml);
    quantity<enzimology::specific_enzyme_activity[U / mg]> specific_activity = activity / protein_concentration;
```

or 


```suggestion
    QuantityOf<enzimology::catalytic_activity> auto units = processed_substrate / time;
    QuantityOf<enzimology::catalytic_activity> auto nanokatals = units[nkat]; // conversion if we want, 1 U is about 16.67 nkat

    QuantityOf<enzimology::volume_activity> auto activity = units / volume;

    QuantityOf<enzimology::protein_concentration> auto protein_concentration = (20.0 * mg) / (1.0 * ml);
    QuantityOf<enzimology::specific_enzyme_activity> auto specific_activity = activity / protein_concentration;
```